test/cql-pytest/test_permissions.py

```diff
@@ -197,13 +197,18 @@ def test_udf_permissions_quoted_names(cassandra_bug, cql):
 # one function with the given name, and that it fails if there are multiple functions with the same name,
 # regardless of the permissions of the user.
 # If the signature is specified, test that the permission check is performed as usual.
-def test_drop_udf_with_same_name(cql):
+# This test is marked cassandra_bug because it exposes a small error-path
```

Comment:
Have you maybe:
1. found an issue opened against Cassandra with this bug?
2. found this behavior documented in Cassandra?
3. in case not, raised such an issue?
I don't mean you should take any action here, I'm just curious how do we handle such events. My current understanding is that we're just aligning scylla to match cassandra.

Reply:
I was too lazy to open a Cassandra issue, but I really should. I'll do it now.
Yes, we should align to Cassandra if there are several "reasonable" options to choose and Cassandra chose one for us. But sometimes, what Cassandra did looks like an obvious bug and if so we shouldn't reproduce it (and yes, it's better to report it to them to see if they don't consider it "a feature, not a bug"). Sometimes - as in this case - the bug is so minor (printing the wrong error message) that it's almost insulting to report such a bug, but maybe we should.

Reply:
Opened two Cassandra issues:
https://issues.apache.org/jira/browse/CASSANDRA-19005
https://issues.apache.org/jira/browse/CASSANDRA-19006

By the way, I didn't remember, but test_permissions.py was already filled with several cassandra_bug cases. I guess this area of Cassandra has a lot of bugs :-)